worker/firewaller/firewaller.go

```diff
@@ -975,24 +975,12 @@ func (fw *Firewaller) updateForRemoteRelationIngress(appTag names.ApplicationTag
 
 	// If there's still too many after merging, look for any firewall whitelist.
 	if cidrs.Size() > maxAllowedCIDRS {
-		cidrs = make(set.Strings)
-		rules, err := fw.firewallerApi.FirewallRules("juju-application-offer")
+		cfg, err := fw.firewallerApi.ModelConfig()
 		if err != nil {
 			return nil, errors.Trace(err)
 		}
-		if len(rules) > 0 {
-			rule := rules[0]
-			if len(rule.WhitelistCIDRS) > 0 {
-				for _, cidr := range rule.WhitelistCIDRS {
-					cidrs.Add(cidr)
-				}
-			}
-		}
-		// No relevant firewall rule exists, so go public.
-		if cidrs.Size() == 0 {
-			cidrs.Add(firewall.AllNetworksIPV4CIDR)
-			cidrs.Add(firewall.AllNetworksIPV6CIDR)
-		}
```

Comment:
This is still needed. If the cidrs size > max allowed and the model config for ingress allow is not set we need the fallback

Reply:
Speaking of which, should the config default actually by "0.0.0.0/0,::/0", which is both v4 and v6 default routes?